packages/tools/devtools/devtools-view/src/components/ContainerSummaryView.tsx

```diff
@@ -213,6 +211,11 @@ export function ContainerSummaryView(props: ContainerSummaryViewProps): React.Re
 		ContainerStateMetadata | undefined
 	>();
 
+	// Set of features supported by the corresponding Container-level devtools instance.
+	const [supportedFeatures, setSupportedFeatures] = React.useState<
```

Comment:
You actually shouldn't need to track this as state here. At the root of the devtools view, we query for this once and save it off in a React Context. You can access them through the `useContainerFeaturesContext` 🙂